autopush/db.py

```diff
@@ -495,7 +495,8 @@ def all_channels(self, uaid):
         # Functions that call store_message() would be required to
         # update that list as well using register_channel()
         try:
-            result = self.table.get_item(consistent=True, uaid=hasher(uaid),
+            result = self.table.get_item(consistent=True,
+                                         uaid=hasher(str(uaid)),
```

Comment:
str again? The type sig indicates its already a str? Does that mean its being used incorrectly somewhere?